lomax/scripts/kill-all.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+while true 
+    do
+        pkill -9 "lomax";
```

Comment:
lomax/scripts/stress-test-1.sh could potentially spawn a lot of lomax instances depending on the argument passed in. This may get out of control if the system can't handle it. The reason I wrote this is because this allows anything "lomax" to be killed without any wait. Is there's a better way to do the above use case?